MonoGame.Framework/WindowsUniversal/UAPGameWindow.cs

```diff
@@ -216,6 +217,27 @@ private void UpdateSize()
             }
         }
 
+        private void Dispatcher_AcceleratorKeyActivated(CoreDispatcher sender, AcceleratorKeyEventArgs args)
+        {
+            // NOTE: Dispatcher event is used becuase KeyDown event doesn't handle Alt key
+            var key = InputEvents.KeyTranslate(args.VirtualKey, args.KeyStatus);
+            switch (args.EventType)
+            {
+                case CoreAcceleratorKeyEventType.KeyDown:
+                case CoreAcceleratorKeyEventType.SystemKeyDown:
+                    if (KeysHelper.IsKey((int)key))
```

Comment:
InputEvents.KeyTranslate should always return a valid key, so this should always be true.